server/www/fountain.js

```diff
@@ -0,0 +1,33 @@
+var of;
+const button = document.querySelector('input');
+const paragraph = document.querySelector('p');
```

Comment:
it doesn't look like u are using this `paragraph` variable, if u don't plan to use it u should delete this line (otherwise it's just extra code adding noise)